### ISSUE TITLE
Game version to 21.0 causes gameplay changes

### ISSUE THREAD

**Issue by Goober5000** [scp-fs2open/fs2open.github.com#3194]
As reported on Discord, MVPs 4.4.0 caused the corvettes in "The Great Hunt" (SM1-06.fs2) to follow their waypoints at a different speed than normal.  This was narrowed down to the game version tag in mv_root-mod.tbm...
```
#GAME SETTINGS
$Target Version:
+Major: 21
+Minor: 0
+Build: 0
```
[mv_root-mod.tbm.txt](https://github.com/scp-fs2open/fs2open.github.com/files/5920745/mv_root-mod.tbm.txt)

As a result, MVPs 4.4.1 do not contain the version tag.  But this is actually a bug with the version tag itself.  One of the features enabled by the target version is causing the corvettes to change speed when they shouldn't.

To fix this bug it will be necessary to inspect the uses of `mod_supports_version` in the codebase and see what causing the unintended side effects.

**Comment by Goober5000** [comment]
A quick look indicates it might be caused by `Fix_ai_class_bug` because the corvettes are both assigned the class of Colonel.  In that case maybe it isn't actually a bug, although I would be surprised if different AI classes are *intended* to follow waypoints at different speeds.

**Comment by Baezon** [comment]
Fixed by #3195
